src/components/GetStarted/Header.tsx

```diff
@@ -1,23 +1,76 @@
-import React, { useCallback } from 'react'
+import React, { ReactNode, useCallback, useState, useEffect } from 'react'
 import { useHistory } from 'react-router-dom'
 // @ts-ignore
 import { GU, IconArrowRight, useTheme, useLayout } from '@aragon/ui'
 import { fontWeight } from '../../style/font'
 import BrandButton from '../BrandButton/BrandButton'
 import LayoutLimiter from '../Layout/LayoutLimiter'
 import { CONVERTER_PATH } from '../../Routes'
+import { useAccountBalances } from '../../providers/AccountBalances'
+
+type BalanceStatus = 'default' | 'success' | 'noMigrationsAvailable'
+
+const MESSAGES: Record<BalanceStatus, ReactNode> = {
+  default: (
+    <>
+      Use Aragon Migrate system to upgrade your ANT balance to the newest
+      version of the token&nbsp;contract.
+    </>
+  ),
+  success: (
+    <>
+      Success!{' '}
+      <span role="img" aria-label="raised-hands">
+        🙌🏼
+      </span>{' '}
+      <span role="img" aria-label="party">
+        🎊
+      </span>{' '}
+      You have migrated all your ANT balance to ANT v2. This account doesn’t
+      hold any more ANT v1 balance. Try a different account.
+    </>
+  ),
+  noMigrationsAvailable: (
+    <>
+      There are no migrations available for this account. Enable a different
+      wallet to check if you have any ANT v1 tokens to migrate.
+    </>
+  ),
+}
 
 function Header({ ...props }: React.HTMLAttributes<HTMLElement>): JSX.Element {
   const history = useHistory()
   const theme = useTheme()
-
+  const { antV1, antV2 } = useAccountBalances()
   const { layoutName } = useLayout()
+  const [balanceStatus, setBalanceStatus] = useState<BalanceStatus>('default')
+
   const compactMode = layoutName === 'small'
 
   const handleNavigateToConverter = useCallback(() => {
     history.push(CONVERTER_PATH)
   }, [history])
 
+  useEffect(() => {
+    if (
+      antV1.balance &&
+      antV1.balance.isZero() &&
+      antV2.balance &&
+      antV2.balance.gt('0')
+    ) {
+      setBalanceStatus('success')
+    }
+
+    if (
+      antV1.balance &&
+      antV1.balance.isZero() &&
+      antV2.balance &&
+      antV2.balance.isZero()
+    ) {
+      setBalanceStatus('noMigrationsAvailable')
+    }
+  }, [antV1.balance, antV2.balance])
```

Comment:
Ahh, we forgot add the fall through so that it resets when switching accounts in the wallet. This would work!

```suggestion
  useEffect(() => {
    if (antV1.balance && antV2.balance) {
      if (antV1.balance.isZero() && antV2.balance.gt('0')) {
        setBalanceStatus('success')

        return
      }

      if (antV1.balance.isZero() && antV2.balance.isZero()) {
        setBalanceStatus('noMigrationsAvailable')

        return
      }
    }

    setBalanceStatus('default')
  }, [antV1.balance, antV2.balance])
```